package.json

```diff
@@ -46,6 +43,19 @@
     "vue-template-compiler": "2.2.6",
     "vue-template-es2015-compiler": "1.3.2"
   },
+  "scripts": {
+    "commit": "git-cz",
+    "commitmsg": "validate-commit-msg",
+    "prepush": "npm run verify",
+    "verify": "npm run style && npm t",
+    "test": "jest",
+    "test:watch": "npm t -- --watch",
+    "test:coverage": "npm t -- --coverage",
+    "test:ci": "npm run test:coverage",
+    "style": "eslint .",
```

Comment:
I can see here the `style` being re-used for verify but why not just directly run together with fix? 

I mean `--fix` is something you wanna perform no matter what result `eslint .` throws, because you just don't wanna push code that has either errors or warns. So of me it's not that big difference, but I might be missing something important. And 2nd in days where ppl use IDEs/Editors I don't expect someone running `eslint .` manually just to run it again with `--fix` or am I wrong?

And another just semantic thing, when I saw `style` for the first time I thought it related to styles, which has common sense for "visual" styles like css, scss. So running `yarn styles` makes me expect styles being preprocessed (scss => csc). Guess, just a point of view. Linting for me does the stuff which is not always related to visual code (style) perception e.g `consistent-return`.